spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -797,8 +793,12 @@ skip_inc_sync("CP/DP labels #" .. strategy, function()
             assert.matches("^(%d+%.%d+)%.%d+", v.version)
             assert.equal(CLUSTERING_SYNC_STATUS.NORMAL, v.sync_status)
             assert.equal(CLUSTERING_SYNC_STATUS.NORMAL, v.sync_status)
-            assert.equal("mycloud", v.labels.deployment)
-            assert.equal("us-east-1", v.labels.region)
+            -- TODO: The API output does include labels and certs when the
+            --       incremental sync is enabled.
```

Comment:
For perfect solution, we need to add a feature for incremental sync: https://konghq.atlassian.net/browse/KAG-5935